showdependencygraph.cgi

```diff
@@ -267,31 +268,15 @@ if ($webdotbase =~ /^https?:/) {
 }
 else {
   # Local dot installation
+  my $image_data;
+  open my $pipe, '-|', "\"$webdotbase\" -Tpng $filename";
```

Comment:
Lets take this opportunity to switch to the open's safer pipe list form and add error handling:
```
open(my $pipe, '-|', $webdotbase, '-Tpng', $filename)
  || ThrowCodeError(...);
```